### PR TITLE
Merge aspnet/WebHooks into this repository

### PR DESCRIPTION
**Pull request by natemcmaster**
Part of https://github.com/aspnet/AspNetCore/issues/3598

This merges the history of https://github.com/aspnet/WebHooks into this repository. The only changes between this PR and [latest commit on master of aspnet/WebHooks](https://github.com/aspnet/WebHooks/tree/9887b4ecbc4a3a79902781c05c30020539a29e91) are:

* removed KoreBuild 
* removed other CI infrastructure bits
* updated the readme and RepositoryUrl values
* reorganized all of aspnet/WebHooks into src/WebHooks/

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -9,11 +9,17 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)eng\AspNetCore.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>7.3</LangVersion>
```
Comment (mkArtakMSFT) [inline Directory.Build.props]: Seems unrelated.

Comment (natemcmaster) [reply Directory.Build.props]: The default is 7.0. KoreBuild overrides the default to 7.2, so I had to add this after removing KoreBuild stuff.
